### ISSUE TITLE
[request.js/dashboard.js] As a user, I should be able to ask a friend for an update

### ISSUE THREAD

**Issue by NathanBland** [whiteboards/converse#40]
## Goals

Create request functionality off of the user cards so that questions can be asked easily.

Questions able to be asked (initially): 
- [ ] What are you up to?
## Acceptance

As a user, I can create a new request for a friend from their user card, and a notification is created for them to respond to that question.
## Stretch

Additional questions can be asked:
- [ ] What are you listening to?
- [ ] What are you reading?
- [ ] What are you watching?
- [ ] Where are you/where are you headed?
- [ ] What are you....? (custom questions)

**Comment by crodeheaver** [comment]
@NathanBland When they hit the ask button, are you thinking it should be brought to a new page or should we dynamically change the content on screen?

**Comment by NathanBland** [comment]
The ask, nudge and pester buttons were just place holder buttons.

I hadn't thought out the next stage.

**Comment by crodeheaver** [comment]
So it's left up to my discretion then?

**Comment by NathanBland** [comment]
Yes, I had wanted at least a quick ask function to be part of the user cards that show up on the dashboard, but I understand that not all of them will fit as well.

**Comment by crodeheaver** [comment]
Well I had been thinking it would be perhaps be a radial button list with the options.

The more I think about this one, the more complex it gets. I'm going to have to put some more thought into the design.

**Comment by NathanBland** [comment]
I think an easy prototype method would be a drop down selection. With a simple precursor question like: ask what..

They are up to
They are watching
They are listening to
Etc.

**Comment by crodeheaver** [comment]
Well what I'm trying to decide is what makes the most sense after that. Dismiss the card or reset the card.

And on that note, where do we get replies? Should I spawn the response card on the right? I'm assuming we won't worry about that too much till we have sockets set up, but it would be good to know.

**Comment by NathanBland** [comment]
Ah, that I might actually be able to shed some light on. 

So, there are a few different ways we can do the action you are talking about. My first thought is to rotate that card to the bottom of their friends "grid" because it is the one they most recently asked for an update from. Beyond this, we should disable the button on that particular friend for x amount of time, or until an update is given. 

This could turn into a user preference setting actually "Only allow people to ask me for updates every x minutes/y hours". I know this starts getting complex, but its a down the road kind of thing. For now I'd say just move the card to the bottom of the list, and maybe disable the button for a few minutes.

As for the response card, those will populate as you thought, on the right side, where the currently hard-coded cards about Keith being up to no good live.
